docs/CreateAKSClusterForScalarProducts.md

```diff
@@ -0,0 +1,274 @@
+# Create an AKS cluster for Scalar products
+
+This document explains how to create an AKS cluster for the Scalar DB and Scalar DL deployment. Please refer to the following document for the step overview to deploy each product.
+
+* [Deploy Scalar DB Server on AKS](./ManualDeploymentGuideScalarDBServerOnAKS.md)
+* [Deploy Scalar DL Ledger on AKS](./ManualDeploymentGuideScalarDLOnAKS.md)
+* [Deploy Scalar DL Ledger and Scalar DL Auditor on AKS](./ManualDeploymentGuideScalarDLAuditorOnAKS.md)
+
+## Steps
+
+You must create an AKS cluster based on the following requirements, recommendations, and your project's requirements. Please refer to the Azure official document ([Azure CLI](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-cli), [PowerShell](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-powershell), or [Azure portal](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-portal)) for the details about how to create the AKS cluster.
+
+## Requirements
+
+There are some requirements for the deployment of Scalar products. Please configure your AKS cluster based on the following requirement and your project's requirements.
+
+### Scalar DB Server
+
+* You must create the AKS cluster with Kubernetes version 1.19 or higher for the Scalar DB deployment.
+
+### Scalar DL Ledger (Ledger only)
+
+* You must create the AKS cluster with Kubernetes version 1.19 or higher for the Scalar DL Ledger deployment.
+
+### Scalar DL Ledger and Auditor (Auditor mode)
+
+* You must create the AKS cluster with Kubernetes version 1.19 or higher for the Scalar DL Ledger and Scalar DL Auditor deployment.
+* You must configure a VNet as follows.
+    * You must connect the **VNet of AKS (for Ledger)** and the **VNet of AKS (for Auditor)** using [VNet Peering](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering). So, you must specify the different IP ranges for the **VNet of AKS (for Ledger)** and the **VNet of AKS (for Auditor)** when you create them.
+    * You must allow the **connections between Ledger and Auditor** to make Scalar DL (Auditor mode) work properly.
+    * Please refer to the [Create network peering for Scalar DL Auditor mode // TODO: Add link of new document]() for more details of these network requirements.
+
+## Recommendations (Optional / Not required)
+
+There are some recommendations for the deployment of Scalar products. These are not required. So, you can choose to apply these recommendations or not based on your requirement.
+
+### Use 4vCPU / 8GB memory nodes for the worker node in the Scalar DL node pool
```

Comment:
We had a short discussion before about this point.

The current document states the concrete `VM type` of each cloud service (e.g., `Standard D2s v3` of Azure and `m5.xlarge` of AWS), but I think it is a dynamic thing. 
For example, there is a possibility that a new type will be released.
So, I describe it based on the node spec (vCPU and Memory) instead of the concrete type name.